src/_guides/language/index.md

```diff
@@ -31,6 +31,10 @@ These two resources are popular with both beginning Dartisans and experts.
 : A list of common questions for those who are interested in stronger
   static typing.
 
+[Sound Dart: Common Problems](/guides/language/common-prob)
+: A list of errors and warnings you may encounter when writing sound
```

Comment:
A list of -> How to fix 

(A little more direct, plus it looks different from the FAQ.)